src/assets/buson.svg

```diff
@@ -0,0 +1,19 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This SVG is added as a raw asset. If the SVG is decorative (you're using it via an <img> with an alt attribute), consider adding a <title> and/or <desc> inside the SVG for better accessibility when the SVG is inlined. Alternatively, if decorative, include aria-hidden="true" on the root or document the usage pattern. Also consider reducing complexity inside the file if it will be inlined as JSX.

**Sugerencia:**
```typescript
Add a <title> (and optional <desc>) to the SVG or mark it aria-hidden if decorative. If this file may be inlined across multiple places, ensure unique ids (see next comment).
```